build.gradle

```diff
@@ -20,6 +17,14 @@ if (build_id != null) {
     version += "+build.${build_id}"
 }
 
+java {
+    toolchain {
+        // Gradle will download the JDK for compiling/running if one isn't found
+        // This should prevent anyone from using an outdated JDK
+        languageVersion.set(JavaLanguageVersion.of(15))
```

Comment:
Gradle projects should not download JDKs, and I don't think its a good idea to restrict what java version a person wishes to compile with.

Reply:
Removed, now the user can compile with whatever JDK they want again.